client/src/pages/CreateContestButton.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { makeStyles, Button, Typography, Grid } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    "font-family" : theme.fontFamily,
+  },
+  button: {
+  	"background-color": "black",
+  	"color": theme.bgcolor,
+    "border-radius": "0",
+    padding: theme.spacing(4, 10)
+  },
+  buttonContainer: {
+  	marginTop: "50px"
```

Comment:
i see a mix of this snake case and string haha. keep consistent, but sometimes if you are in need to write a bit more complicated CSS, you will have to use strings :)